eldritch/events.py

```diff
@@ -2006,22 +2058,41 @@ def BinaryChoice(
 
 class ItemChoice(ChoiceEvent):
 
-  def __init__(self, character, prompt):
+  def __init__(self, character, prompt, decks=None, item_type=items.Item):
```

Comment:
Why do you use both decks and item_type here? I think if you know the deck, then you automatically know the item type, yes? The exception might be weapons and tomes, but I would prefer those be done with getattr(pos, "item_type").
That being said, you don't seem to be using it anywhere, so maybe you can remove it?

Reply:
Weapons and Tomes were what I was thinking of, not sure whether there actually is an encounter that would rely on that feature.  Updated the code at L2085 to use getattr.